src/payment/strategies/ppsdk/step-handler/continue-handler/redirect/RedirectionState.ts

```diff
@@ -25,10 +24,10 @@ export class RedirectionState {
     }
 
     private currentUrlHasRedirectingParam(): boolean {
-        return paramRegex.test(this.getCurrentUrl().search);
+        return PENDING_REDIRECT_PARAM in this.getCurrentUrl().query;
```

Comment:
The result of `this.getCurrentUrl().query` is an object created via the `Object.create(null)` method, so does not have `hasOwnProperty` or any other prototypal methods - hence the `in`